### ISSUE TITLE
Differences between `model.py` and `model_search.py`

### ISSUE THREAD

**Issue by bkj** [quark0/darts#10]
I noticed a couple of discrepencies between `cnn/model.py` and `cnn/model_search.py`:

- https://github.com/quark0/darts/blob/master/cnn/model_search.py#L18 - extra batchnorm
- https://github.com/quark0/darts/blob/master/cnn/model_search.py#L32 - affine=False in stem 
- no auxiliary head in model_search
- no drop_path in model_search

Are there particular reasons for these choices (particularly the first two)?  Or are they bugs/arbitrary decisions?

~ Ben

__EDIT:__ Same question applies to different choices in hyperparameters in the search vs. final training (batch size, init_channels, etc)

**Comment by quark0** [comment]
These decisions are not arbitrary. It's useful to have batch norms for all candidate ops (with learnable affine parameters disabled) to ensure their outputs are comparable during architecture search, which would make it easier for the algorithm to figure out the relative importance of each op. Auxiliary towers and pathwise dropout are additional augmentations that are only enabled during final evaluation.

**Comment by bkj** [comment]
Got it, thanks.  Do you think it's bad to use the extra batchnorms and `affine=False` mentioned above in final training? 

Thanks
~ Ben

**Comment by quark0** [comment]
Not necessarily, but it's good to stay with common practices for fair comparisons with other models.
